app/Http/Controllers/IgnoreProjectController.php

```diff
@@ -8,8 +8,13 @@ class IgnoreProjectController extends Controller
 {
     public function __invoke(Project $project)
     {
-        $project->update(['ignored' => true]);
-
+				$project->update(['ignored' => true]);
+				
+				if (request()->wantsJson()) {
+					return response()->json([
+						'success' => true
+					], 200);
+				}
```

Comment:
Instead of returning `'success' => true`, let's just return the positive status code.

`response()->json([], 200)` should do it I think.

Reply:
Fixed